Source/WebCore/dom/ContainerNode.h

```diff
@@ -161,10 +163,10 @@ class ContainerNode : public Node {
     void executePreparedChildrenRemoval();
     enum class DeferChildrenChanged : bool { No, Yes };
     enum class DidRemoveElements : bool { No, Yes };
-    DidRemoveElements removeAllChildrenWithScriptAssertion(ChildChange::Source, NodeVector& children, DeferChildrenChanged = DeferChildrenChanged::No);
+    DidRemoveElements removeAllChildrenWithScriptAssertion(ChildChange::Source, NodeVector& children, DeferChildrenChanged = DeferChildrenChanged::No, CanDelayNodeDeletion = CanDelayNodeDeletion::No);
```

Comment:
Why do we need to use CanDelayNodeDeletion::No? What test case / scenario requires that?